windows/src/engine/keyman32/tests/keyman-engine-tests/pch.cpp

```diff
@@ -1,6 +1,53 @@
-//
-// pch.cpp
-// Include the standard header and generate the precompiled header.
-//
-
+/**
```

Comment:
All this really shouldn't be in pch.cpp, which should only be a stub for precompiled headers.

Reply:
> All this really shouldn't be in pch.cpp, which should only be a stub for precompiled headers.

Ok I will move it all out to gtest_main.cpp as that makes it clear what it really is after all. I could go with gtest_mem as the original authour did but `gtest_main` is even clearer.
I will revert pch.cpp to just being a stub.